test/rpc_test.py

```diff
@@ -1091,6 +1091,51 @@ def test_call_method_on_rref(self):
 
         self.assertEqual(result, sum(vals))
 
+    def _test_rref_leak(self, ignore_leak=False):
+        rpc.init_rpc(
+            name="worker{}".format(self.rank),
+            backend=self.rpc_backend,
+            init_method=self.init_method,
+            rank=self.rank,
+            world_size=self.world_size,
+            rpc_agent_options=self.rpc_agent_options,
+        )
+
+        # This is for the below `dist.barrier`.
+        # For `RpcAgent` other than `ProcessGroupAgent`,
+        # no `_default_pg` is initialized.
+        if not dist.is_initialized():
```

Comment:
I added a `_initialize_pg` helper method in `dist_autograd_test.py` for this. Maybe we can move that to dist_utils.py and use it in both tests?

Reply:
Will do dedup in a followup PR